config/migrations/20241213190000-beleidsvelden-themis-concepts.ttl

```diff
@@ -0,0 +1,7573 @@
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix schema: <http://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<http://themis.vlaanderen.be/id/beleidsveld/31adc08e-0806-4f72-9b3f-8d3ecbc65579> a skos:Concept ;
+    mu:uuid "31adc08e-0806-4f72-9b3f-8d3ecbc65579" ;
+    prov:generatedAtTime "2024-06-21T00:00:00"^^xsd:dateTime ;
+    schema:position 1;
+    skos:broader <http://themis.vlaanderen.be/id/beleidsdomein/be302e1b-f4d8-4212-a67d-bf992e6effcf> ;
+    skos:inScheme <http://themis.vlaanderen.be/id/concept-scheme/0012aad8-d6e5-49e2-af94-b1bebd484d5b> ;
+    skos:prefLabel "Luchtgebonden mobiliteit en transport"@nl ;
+    skos:topConceptOf <http://themis.vlaanderen.be/id/concept-scheme/0012aad8-d6e5-49e2-af94-b1bebd484d5b> .
+
+<http://themis.vlaanderen.be/id/beleidsveld/a5cf604b-b612-4a40-9741-4f0b78e79253> a skos:Concept ;
+    mu:uuid "a5cf604b-b612-4a40-9741-4f0b78e79253" ;
+    prov:generatedAtTime "2024-06-21T00:00:00"^^xsd:dateTime ;
+    schema:position 4;
+    skos:broader <http://themis.vlaanderen.be/id/beleidsdomein/be302e1b-f4d8-4212-a67d-bf992e6effcf> ;
+    skos:inScheme <http://themis.vlaanderen.be/id/concept-scheme/0012aad8-d6e5-49e2-af94-b1bebd484d5b> ;
+    skos:prefLabel "Weggebonden mobiliteit en transport"@nl ;
+    skos:topConceptOf <http://themis.vlaanderen.be/id/concept-scheme/0012aad8-d6e5-49e2-af94-b1bebd484d5b> .
+
+<http://themis.vlaanderen.be/id/beleidsveld/39ca6f6c-e7e2-4a74-9ba9-2cdc679d8237> a skos:Concept ;
+    mu:uuid "39ca6f6c-e7e2-4a74-9ba9-2cdc679d8237" ;
+    prov:generatedAtTime "2024-06-21T00:00:00"^^xsd:dateTime ;
+    schema:position 5;
+    skos:broader <http://themis.vlaanderen.be/id/beleidsdomein/be302e1b-f4d8-4212-a67d-bf992e6effcf> ;
+    skos:inScheme <http://themis.vlaanderen.be/id/concept-scheme/0012aad8-d6e5-49e2-af94-b1bebd484d5b> ;
+    skos:prefLabel "Watergebonden mobiliteit en transport"@nl ;
+    skos:topConceptOf <http://themis.vlaanderen.be/id/concept-scheme/0012aad8-d6e5-49e2-af94-b1bebd484d5b> .
+
+<http://themis.vlaanderen.be/id/beleidsveld/8509d78a-c468-4bfa-9226-cb75782a7864> a skos:Concept ;
+    mu:uuid "8509d78a-c468-4bfa-9226-cb75782a7864" ;
+    prov:generatedAtTime "2024-01-01T00:00:00"^^xsd:dateTime ;
```

Comment:
invalidation?

Reply:
No, as specified in the ticket, we can get away with invalidating only the 3 fields we renamed because none of the structure w.r.t. government domains changed

Reply:
fine by me